lib/cache/cache.js

```diff
@@ -0,0 +1,20 @@
+var cacheManager = require('cache-manager')
+
+class Cache {
+  constructor () {
+    this.cache = null
+
+    switch (process.env.CACHE_STORAGE) {
+      case 'redis':
+        var redisStore = require('cache-manager-ioredis')
+        this.cache = cacheManager.caching({ store: redisStore, url: process.env.REDIS_URL, refreshThreshold: 3 })
```

Comment:
should `refreshThreshold` be a configurable variable?

Reply:
Hey you are right, I've moved every variable to environment variables. Defaults has been defined by the parent package already,

```
max: 500
ttl: null
```